### ISSUE TITLE
Aliases are always returned as mocked data

### ISSUE THREAD

**Issue by zofiag** [APIs-guru/graphql-faker#89]
Even though my types exist in the backend and are not mocked in the SDL file, graphql-faker returns it as mocked data when aliases are used. Example:

1. doesn't work - returns mocks instead of real data
```
query Test {
  verifiedStates: dictionaries(types: [VERIFIED_STATES]) {
      items {
        label
        value
      }
   }
}
``` 
2. works - returns real data
```
query Test {
  dictionaries(types: [VERIFIED_STATES]) {
      items {
        label
        value
      }
   }
}
```

**Comment by IvanGoncharov** [comment]
@zofiag Thanks for reporting 👍 
Fixed in 71b5461 and released as `graphql-faker@2.0.0-rc.14` 🎉 
Can you please test it?

**Comment by zofiag** [comment]
it's working! thanks for a quick fix 🎉
